components/script/dom/htmliframeelement.rs

```diff
@@ -457,6 +457,24 @@ impl HTMLIFrameElementMethods for HTMLIFrameElement {
     make_getter!(FrameBorder, "frameborder");
     // https://html.spec.whatwg.org/multipage/#other-elements,-attributes-and-apis:attr-iframe-frameborder
     make_setter!(SetFrameBorder, "frameborder");
+
+    // https://html.spec.whatwg.org/multipage/#dom-iframe-name
+    fn SetName(&self, name: DOMString) {
+        if let Some(window) = self.GetContentWindow() {
+            let mut child_window = &*window;
```

Comment:
Is this line actually necessary? If it is, we should at least be able to remove the `mut`.